partials/transactions/txListType/contractCallTx.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div class="transaction">
```

Comment:
i think we need to do analysis on the `txType` components, they are very similliar, and as mentioned above we can delegate much of the logic to specific components like you are doing very nicely in the `txTypeLabel` component